src/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,149 @@
+/** @jsxImportSource theme-ui */
+import { get } from "@theme-ui/css"
+import React, { useCallback, useState } from "react"
+import { useTabContext } from "./context"
+import { TabContext } from "./context"
+
+export interface TabProps {
+  initialOpenIndex?: number
+  children: React.ReactNode
+  activeBackgroundColor?:
+    | "primary"
+    | "secondary"
+    | "text"
+    | "success"
+    | "warning"
+}
+
+const Tab = ({
+  children,
+  initialOpenIndex,
+  activeBackgroundColor = "secondary",
+}: TabProps) => {
+  const [openIndex, setOpenIndex] = useState<number | undefined>(
+    initialOpenIndex
+  )
+
+  const toggleOpen = useCallback(
+    (index: number) => {
+      if (openIndex === index) setOpenIndex(undefined)
+      else setOpenIndex(index)
+    },
+    [openIndex]
+  )
+
+  const allItems = React.Children.toArray(children)
+    .filter((child) => {
+      return React.isValidElement(child)
+    })
+    .map((child, index) => (
+      <TabContext.Provider
+        value={{ index, toggleOpen, openIndex, activeBackgroundColor }}
+        key={index}
+      >
+        {child}
+      </TabContext.Provider>
+    ))
+
+  const tabLink = allItems.filter((item) =>
+    item.props.children.props.hasOwnProperty("title")
+  )
+  const tabPanel = allItems.filter((item) =>
+    item.props.children.props.hasOwnProperty("index")
+  )
+  return (
+    <>
+      <div sx={{ display: "flex", flexDirection: "row" }}>
+        {tabLink.map((item) => item)}
+      </div>
+      <div>{tabPanel.map((item) => item)}</div>
+    </>
+  )
+}
+
+export interface TabLinkProps {
+  title: string
+  // children: React.ReactNode
+  // className?: HTMLAttributes<HTMLDivElement>["className"]
+}
+
+const TabLink = ({ title }: TabLinkProps) => {
+  const { index, toggleOpen, openIndex, activeBackgroundColor } =
+    useTabContext()
+  const isOpen = openIndex === index
+  return (
+    <div
+      role="button"
+      aria-disabled="false"
+      aria-expanded={isOpen}
+      sx={{
+        px: 6,
+        py: 2,
+        variant: "text.body1",
```

Comment:
This text variant is used here, but the only place where a text inside this element is in the title span, where another text variant is used and overrides this one